### ISSUE TITLE
[Bug] variable inside another variable

### ISSUE THREAD

**Issue by Gabrielsc** [yesoreyeram/grafana-infinity-datasource#307]
**Describe the bug**
I don't know if it would be exactly a bug. For me, using the grafana variables, one being referenced within the plugin option "HTTP method, Query param, Headers" it does not identify the value of the variable.

For example:
I have the following variables configured:
variable 1: "from" with value "$from_jan"
variable 2: "to" with value "$to_jan"
variable 3: "from_jan" with value "01-01"
variable 4: "to_jan" with value "31-01"

If you insert within the plugin options "HTTP method, Query param, Headers" a POST type query calling a variable for example $from_jan with the value "01-01", it will read the value. But if you use the $from variable that has the value "$from_jan" (ie it calls the other variable), then the Infity plugin doesn't read the value.

**Screenshots**
![infinity](https://user-images.githubusercontent.com/8128226/166343085-8d191b52-0fcd-4bf1-8c3e-e3bc4ffbf023.png)

**Version Details:**

- Grafana version : 8.5.1
- Plugin version : 0.8.8

**Comment by yesoreyeram** [comment]
I don't think this is a plugin problem. If i read this correctly, the variable provider for $from sould be responsible for interpolating this.

You can confirm this by using text panel and check if the variable renders correctly the there. ( To confirm that this is not a plugin issue )

**Comment by Gabrielsc** [comment]
Thanks for the feedback. It was just the test I did about testing.
Below we have the variables.
![image](https://user-images.githubusercontent.com/8128226/166939396-8f8b0dba-1d66-444f-ba1e-1fcf014ce371.png)
![image](https://user-images.githubusercontent.com/8128226/166939457-228997bf-7384-4431-b817-5438519a060d.png)

Panel configuration:
![image](https://user-images.githubusercontent.com/8128226/166939683-88d9bb5f-55db-4936-9d84-3c2ead4ea3df.png)

Dashboard view:
![image](https://user-images.githubusercontent.com/8128226/166939064-a2f7c535-7e3a-4819-8213-344b29471491.png)

That is, from what I understand, grafana identifies what for example $from has the value $from_abr configured:
![image](https://user-images.githubusercontent.com/8128226/166940599-c4dcff68-9ed4-458a-b844-fc506e7d1adf.png)

Once again thank you for your attention, sorry for the google translate english.

**Comment by Gabrielsc** [comment]
> I don't think this is a plugin problem. If i read this correctly, the variable provider for $from sould be responsible for interpolating this.
> 
> You can confirm this by using text panel and check if the variable renders correctly the there. ( To confirm that this is not a plugin issue )

test done above
